### DISCUSSION TITLE
Please read before posting

### DISCUSSION THREAD

**Discussion by macraig** [meilisearch/product#754] [News]
# Welcome to the Product Discussions Space

Welcome to our GitHub Discussions space! This is your go-to place for engaging in conversations about our product and sharing your valuable feedback.

## What to post here:
- **Product Discussions:** Share your thoughts, ideas, and suggestions about the product.
- **Product Feedback:** Let us know what you love, what you'd like to see improved, and any new features you'd like us to consider.

## What NOT to post here:
- **Support Questions:** For any support-related inquiries, please use our [Discord community](https://discord.gg/VJq2T4tEBk).
- **Bug Reports:** To report bugs, please use the [Meilisearch issue tracker](https://github.com/meilisearch/Meilisearch/issues) or the appropriate issue tracker for the specific repository.

Thank you for helping us keep this space focused and productive! Your contributions are invaluable to our community.

**Comment by gizmomelb** [comment]
request:  meillisearch to index lyric .lrc files (used by Jellyfin media streamer)- then you can find songs by lyrics.
